### ISSUE TITLE
alt attributes to necessary images

### ISSUE THREAD

**Issue by lukehawranick** [WVU-CS230-2023-01-Group02/bottomText#79]
# Alt Text

### Description of Bug
There are several instances of images throughout the site which do not have alt text. These attributes are important, not only for accessibility purposes (ex: reading document content aloud, displays on failed image load) but also for SEO. This pertains to all non-decorative images, and the message should be short yet descriptive.

### Expected behavior
In the event that an image fails to load, these alt text attributes should display. 

#### Specific Instances
- Layouts/home/home.component.html (1)
- userPage/tabs/account-tab/account-tab.component.html (2)
- userPage/tabs/university-tab/university-tab.component.html (1)
- Navigation/Navbar/Navbar.component.html (1)
